ivy/array/elementwise.py

```diff
@@ -1243,6 +1243,29 @@ def log2(self: ivy.Array, *, out: Optional[ivy.Array] = None) -> ivy.Array:
             an array containing the evaluated base ``2`` logarithm for each element
             in ``self``. The returned array must have a real-valued floating-point
             data type determined by :ref:`type-promotion`.
+
+	Examples
+        --------
+
+        Using :code:`ivy.Array` instance method:
```

Comment:
Could you provide a simple example by only using a 0-dim input? for example `ivy.array(10)`